README.md

```diff
@@ -23,35 +25,56 @@ npm install
 ```
 
 ### npm
-This module is not on npm yet.
+
+```sh
+npm install node-firefox-start-simulator
+```
 
 ## Usage
 
 #### Callback
 
 ```javascript
-var start = require('./node-firefox-start-simulator');
-start(function(err, sim) {
-
-})
+var startSimulator = require('node-firefox-start-simulator');
+startSimulator(options) // returns a Promise
+  .then(function(simulator) {
+  });
```

Comment:
This being indented feels a bit strange, but then I see how it's hard to do with that promise comment. Might be easier to read as:

```javascript
// `startSimulator` returns a Promise
startSimulator(options).then(function(simulator) {
});
```

Pretty nitpicky though; if you disagree just don't change it :wink: